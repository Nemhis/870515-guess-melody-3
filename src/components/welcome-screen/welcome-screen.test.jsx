import React from 'react';
import renderer from 'react-test-renderer';

import App from './welcome-screen.jsx';

const name = `Boss`;

describe(`<Welcome-screen>`, () => {
  it(`Render <Welcome-screen>`, () => {
    const tree = renderer
      .create(<App/>)
      .toJSON(`test`);

    expect(tree).toMatchSnapshot();
  });

  it(`Render <Welcome-screen> with name`, () => {
    const tree = renderer
      .create(<App
        name={name}
      />)
      .toJSON(name);

    expect(tree).toMatchSnapshot();
  });
});
