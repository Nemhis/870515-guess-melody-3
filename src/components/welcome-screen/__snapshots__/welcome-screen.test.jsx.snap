// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Welcome-screen> Render <Welcome-screen> 1`] = `
<p>
  Hello, 
  Stranger
  .
</p>
`;

exports[`<Welcome-screen> Render <Welcome-screen> with name 1`] = `
<p>
  Hello, 
  Boss
  .
</p>
`;
